examples/problems/200_custom_cost_matrices.ipynb

```diff
@@ -117,7 +117,7 @@
     {
```

Comment:
<p>also print / check the other ones here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>